ajax/libs/opus-recorder/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "opus-recorder",
+  "version": "4.1.0",
+  "description": "A library for recording opus encoded audio",
+  "homepage": "https://github.com/chris-rudmin/opus-recorder",
```

Comment:
Let's remove this property, as it's the same as `repository` property, we don't need to maintain both of them ;)